yt_dlp/extractor/itv.py

```diff
@@ -87,15 +82,64 @@ def _real_extract(self, url):
                 },
                 'variantAvailability': {
                     'featureset': {
-                        'min': ['hls', 'aes', 'outband-webvtt'],
-                        'max': ['hls', 'aes', 'outband-webvtt']
+                        'min': featureset,
+                        'max': featureset
                     },
-                    'platformTag': 'dotcom'
+                    'platformTag': platform_tag
                 }
-            }).encode(), headers=headers)
-        video_data = ios_playlist['Playlist']['Video']
-        ios_base_url = video_data.get('Base')
+            }).encode(), headers=headers, fatal=fatal)
 
+    def _get_subtitles(self, video_id, variants, ios_playlist_url, headers, *args, **kwargs):
+        subtitles = {}
+        platform_tag_subs = None
+        featureset_subs = None
+        for platform_tag, featuresets in variants.items():
+            for featureset in featuresets:
+                if (isinstance(featureset, list) and len(featureset) >= 3
+                        and featureset[2] == 'outband-webvtt'):
```

Comment:
I believe this is same as 
```suggestion
                if try_get(featureset, lambda x: x[2]) == 'outband-webvtt':
```